src/NUnitConsole/nunit3-console/nunit3-console.csproj

```diff
@@ -25,6 +25,7 @@
     <WarningLevel>4</WarningLevel>
     <Externalconsole>true</Externalconsole>
     <Commandlineparameters>net-4.5/nunit.framework.tests.dll --wait --inprocess</Commandlineparameters>
+    <LangVersion>3</LangVersion>
```

Comment:
Could we add this limitation on all our projects to guarantee a correct compilation?